index.html

```diff
@@ -240,7 +240,24 @@ <h2>Biryani</h2> <br>
         </div>
       </div>
     </div>
-    <br><br><br>
+    <div class="mySlides">
+      <div class="carousel-box">
+        <div class="carousel-text">
+          <div class="carousel-heading">
+            <h2><i>Curd Rice</i></h2> <br>
+          </div>
+
+          <p class="review">
+            <b>Curd rice, also called yogurt rice, is a dish originating from India. The word "curd" in Indian English refers to unsweetened probiotic yogurt.</b>
+          </p>
+        </div>
+
+        <div class="carousel-img">
+          <img src="https://www.indianhealthyrecipes.com/wp-content/uploads/2022/02/curd-rice-thayir-sadam.jpg" alt="" />
```

Comment:
Don't use external links in src , upload the file to the images folder and link it.
- then commit the changes and push again